packages/client/src/components/form/FetchButton.tsx

```diff
@@ -67,16 +69,12 @@ const ModalContent = styled.div`
   position: relative;
 `
 
-const Heading = styled.div`
-  color: ${({ theme }) => theme.colors.copy};
+const Heading = styled(Text)`
```

Comment:
`<Text>` actually has `align` parameter so for example this `Heading` is not needed if refactored to use Text